packages/iobio-react-components/src/utils/webComponentTools.mts

```diff
@@ -50,3 +52,51 @@ export const setElementStyles = (element: Element, styles: string) => {
 		element.shadowRoot.adoptedStyleSheets = [elementStyles];
 	}
 };
+
+/** File Strategy & Bed URL */
+const fileStrategies = ['WGS', 'WXS', 'ChipSeq', 'RNA-Seq'];
+type FileStrategyKey = (typeof fileStrategies)[number];
+export type DefaultBedUrls = Record<FileStrategyKey, string>;
+
+/**
+ * GitHub Bed File URLs for requesting Bed files from a browser
+ * Iobio Data Broker requires a Bed URL argument
+ * In the browser, relative file path solutions are more complicated
+ */
+// TODO: Finalize URLs after merge, these are just for testing
+const bedShuffled1BrowserUrl =
+	'https://raw.githubusercontent.com/overture-stack/iobio-components/29a2f8ec57ea8d38a00e998c52487b5aafe5095d/packages/iobio-react-components/src/utils/bedFiles/1k_flank_hg38_shuffled1.bed';
+const bedShuffled2BrowserUrl =
+	'https://raw.githubusercontent.com/overture-stack/iobio-components/29a2f8ec57ea8d38a00e998c52487b5aafe5095d/packages/iobio-react-components/src/utils/bedFiles/1k_flank_hg38_shuffled2.bed';
+const bedIlluminaBrowserUrl =
+	'https://raw.githubusercontent.com/overture-stack/iobio-components/29a2f8ec57ea8d38a00e998c52487b5aafe5095d/packages/iobio-react-components/src/utils/bedFiles/hg38_Twist_Bioscience_for_Illumina_Exome_2.5.subset.bed';
+
+export const defaultBedUrls: DefaultBedUrls = {
+	WGS: bedShuffled1BrowserUrl,
+	WXS: bedShuffled2BrowserUrl,
+	ChipSeq: bedShuffled2BrowserUrl,
+	'RNA-Seq': bedIlluminaBrowserUrl,
+};
+
+/** Lookup Default Bed File
+ * @param fileStrategy String file strategy designation for Bed file lookup
+ * @returns bedFileUrl - string
+ */
+export const getDefaultBedFileUrl = (fileStrategy: string | undefined): string | undefined => {
+	const isValidStrategy = fileStrategy && fileStrategies.includes(fileStrategy);
+	if (!isValidStrategy) {
+		console.error('File Strategy is not valid, cannot lookup recommended Bed file');
```

Comment:
There are likely other valid file strategies in someone else's data set, perhaps we want this message to say "no default BED file available for this file strategy" or similar?

Reply:
Updated